english/dream/fly/space/space.md

```diff
@@ -6,6 +6,17 @@ So you head off to galaxy far, far away....
 
 You eventually reach a solar system with a planet that has intelligent life.
 
-While you are there, they explain that dreams are actually the real Universe 
+While you are there, they explain that dreams are actually the real Universe
 and our reality is a delusion we've created to give ourselves a way to escape
 to a more consistent existence.
+
+You almost buy this malarkey when you notice the small one trying to sneak
+your wallet on the sly. Of all the...!
+
+Do you:
+
+[Show these aliens who they're messing with](english/appear-on-falkors-back/falkor.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.